src/test/java/net/sf/jabref/logic/formatter/FormatterTest.java

```diff
@@ -85,6 +85,8 @@ public FormatterTest(Formatter formatter) {
                 new Object[]{new SuperscriptFormatter()},
                 new Object[]{new UnitFormatter()},
                 new Object[]{new RemoveBracesFormatter()}
+                // The EraseFormatter is special since it always returns null to indicate that the field should be deleted.
+                // new Object[]{new EraseFormatter()}
```

Comment:
Not really following the discussion, but it struck me that a functionality that removes all empty fields, i.e., where the field is `""` (or even whitespace) could be useful...